.travis.yml

```diff
@@ -7,25 +7,19 @@ matrix:
     - go: 1.11.x
       env: GO111MODULE=off
     - go: 1.11.x
-      env:
-      - GO111MODULE=on
-      - VET=1
+      env: GO111MODULE=on
     - go: 1.12.x
       env: GO111MODULE=off
     - go: 1.12.x
       env: GO111MODULE=on
     - go: 1.13.x
       env:
       - GO111MODULE=on
-      - GOPROXY=direct
+      - VET=1
```

Comment:
Why vet 1.13, why not the latest version of go?

Reply:
Sorry, I forgot to answer this one before merging. I've been sticking with versions that we happen to use internally, to make it easier to repro on our work laptops.